src/benchmarklib/table_generator.hpp

```diff
@@ -14,25 +14,31 @@ class Table;
 enum class DataDistributionType { Uniform, NormalSkewed, Pareto };
 
 struct ColumnDataDistribution {
-  static ColumnDataDistribution make_uniform_config(double min, double max) {
+  static ColumnDataDistribution make_uniform_config(double min, double max,
+                                                    const std::optional<float>& null_value_ratio = {}) {
```

Comment:
Instead of using an empty default-optional, can't we set it to `flaot null_value_ratio = 0f`?

Reply:
this was for the presentation, reverted it